compute/cloud-client/src/test/java/compute/Util.java

```diff
@@ -217,6 +218,22 @@ && isCreatedBeforeThresholdTime(disk.getCreationTimestamp())) {
     }
   }
 
+  // Delete regional disks which starts with the given prefixToDelete and
+  // has creation timestamp >24 hours.
+  public static void cleanUpExistingRegionalDisks(
```

Comment:
I remember seeing this method in another PR as well. Kindly ensure to resolve merge conflicts in the PRs before merging.